src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md

```diff
@@ -846,6 +890,28 @@ We have fixed hundreds of issues in the Magento 2.3.4 core code.
 
 *  Magento no longer throws an error when you open an image from the product image gallery from the storefront product detail page. Previously, Magento returned this JavaScript error in the console: `TypeError: The expression cannot be converted to return the specified type`.
 
+<!--- MAGETWO-99401-->
+
+*  Order-related `save_after_commit` callbacks are now called for guest checkouts as expected. Previously, the corresponding `order _save_commit_after` events were not dispatched.
+
+<!--- MC-17332-->
+
+*  The product counter and page lister on **Catalog** > **Products** now works correctly after the **Add Store Code to Urls** setting has been enabled or disabled.
+
+<!--- MC-19250-->
+
+*  The new CONSUMERS_WAIT_FOR_MAX_MESSAGES environment variable lets merchants configure how consumers process messages from the message queue. You can set this variable by adding the `consumers-wait-for-messages` option to `bin magento queue:consumers:start`. Using this option can help Magento deployments avoid being affected by long delays in message queue processing. See [Start message queue consumers](https://devdocs.magento.com/guides/v2.3/config-guide/mq/manage-message-queues.html).
```

Comment:
I thought the environment variable was for Cloud, while on-premise users specify a new argument?

Reply:
This is the text that Margaret worked on with the developer, so I will not change it. Will follow up off-Git